elytron/src/main/resources/schema/wildfly-elytron_15_0.xsd

```diff
@@ -1850,6 +1850,20 @@
                         </xs:documentation>
                     </xs:annotation>
                 </xs:attribute>
+                <xs:attribute name="credential-store" default="null" type="xs:string">
```

Comment:
Just a minor comment, we tend not to include `default='null'`.

Reply:
removed